forwarder.go

```diff
@@ -0,0 +1,70 @@
+package k8sportforward
+
+import (
+	"net/http"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/transport/spdy"
+)
+
+type ForwarderConfig struct {
```

Comment:
The name changed. We have to consider this when updating vendor deps.

Reply:
Are we aware where this is vendored? Would be cool to update vendors preemptively there before some unfortunate soul wanders into this.

Reply:
Or is it just `helmClient`?

Reply:
Definitely `helmclient` and I sort this one. There shouldn't be 10 more. So. Fine.